src/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = async function({
   output,
   type,
   content,
-  quality = 80, // only applicable for jpg
+  quality = undefined,
```

Comment:
The default value is already `undefined` we don't need to set it explicitly.

```suggestion
  quality,
```